### PR TITLE
Feature Enhancement: --stdout option added to Output result to STDOUT

### PR DESCRIPTION
**Pull request by rahul26goyal**
conflicts removed

### CODE REVIEW

File: bin/file.js [hunk 1]
```diff
@@ -32,7 +32,12 @@ module.exports = function (program, file) {
   var transformer = new Transformer({transformers: transformers});
   transformer.readFile(file[0]);
   transformer.applyTransformations();
-  transformer.writeFile(program.outFile);
+  if(program.stdout){
```
Comment (dbkaplun) [inline bin/file.js]: Can we change this to `if(program.outFile === '-'){`, as is standard convention for stdout in Unix?

File: bin/file.js [hunk 2]
```diff
@@ -32,7 +32,12 @@ module.exports = function (program, file) {
...
+    transformer.writeToStdout();
+  }
+  else {
+    transformer.writeFile(program.outFile);
+    console.log('The file "' + program.outFile + '" has been written.');
```
Comment (dbkaplun) [inline bin/file.js]: Change to `console.error`

**Comment by nene** [comment]
I would cast my vote against using any specific command line options for STDIN/STDOUT in general.

Lebab should simply read STDIN when no input file given and write to STDOUT when no output file given.

**Comment by aziz** [comment]
@nene totally agree. 
>  Lebab should simply read STDIN when no input file given and write to STDOUT when no output file given.

**Pull request closed without merging**